src/task/mod.rs

```diff
@@ -167,6 +178,9 @@ pub struct Execute {
 
     /// The working directory for the command relative to the root of the project.
     pub cwd: Option<PathBuf>,
+
+    /// A list of environment variables to set before running the command
+    pub env: Option<HashMap<String, String>>,
```

Comment:
As mentioned above, I wonder if we could (should) use an IndexMap in order to do something like

```
env = { BLA="test", FOO="$BLA + $BLA" }
```